### PR TITLE
[MWAN] Updated Gateway tutorial

### PR DESCRIPTION
**Pull request by marciocloudflare**
Addresses PCX-5947

### CODE REVIEW

File: content/magic-wan/tutorials/secure-web-gateway.md [hunk 1]
```diff
@@ -7,68 +7,82 @@ meta:
 
 # Connect to Cloudflare Gateway with Magic WAN
 
-You can route traffic through Magic WAN and [Cloudflare Gateway](/cloudflare-one/policies/filtering/) to secure Internet browsing and SaaS application access from internal users and devices.
+You can route traffic through Magic WAN and filter it with [Cloudflare Gateway](/cloudflare-one/policies/filtering/). Cloudflare Gateway allows you to set up policies to inspect outbound traffic to the Internet through DNS, network, HTTP and egress filtering. Each policy serves different use cases, and we recommend that you [read the Gateway documentation](/cloudflare-one/policies/filtering/) to learn more.
```
Comment (angelampcosta) [inline content/magic-wan/tutorials/secure-web-gateway.md]:
```suggestion
You can route traffic through Magic WAN and filter it with [Cloudflare Gateway](/cloudflare-one/policies/filtering/). Cloudflare Gateway allows you to set up policies to inspect outbound traffic to the Internet through DNS, network, HTTP and egress filtering. Each policy serves different use cases, and we recommend that you read the [Gateway documentation](/cloudflare-one/policies/filtering/) to learn more.
```

File: content/magic-wan/_partials/_traceroute.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ _build:
   list: never
 ---
 
-## Run `traceroute`
+{{<Aside type="note" header="​​Run <code>traceroute</code>">}}
 
 Magic WAN clients connecting through [GRE, IPsec](/magic-wan/get-started/configure-tunnels/), [CNI](/network-interconnect/) or [WARP](/magic-wan/tutorials/warp/) that want to perform a `traceroute` to an endpoint behind a [Cloudflare Tunnel](/cloudflare-one/connections/connect-apps/) will need to change some settings to make the command useful. Refer to [Run `traceroute`](/magic-wan/how-to/traceroute/) for more information.
```
Comment (angelampcosta) [inline content/magic-wan/_partials/_traceroute.md:10]: Just to double check, it's the same link for GRE and IPsec?

File: content/magic-wan/tutorials/secure-web-gateway.md [hunk 3]
```diff
@@ -7,68 +7,82 @@ meta:
...
-5.  If your test was successful, select **Add routes** when you are done.
+{{<Aside type="note">}}
+When you create a GRE tunnel the TTL and MTU fields are auto-populated. The default MTU value of 1476 bytes takes into account the GRE encapsulation overhead of 24 bytes (20 bytes for the outer IP header plus 4 bytes for the mandatory GRE header) that will be added to the original (inner) IP packet when they are sent over the GRE tunnel. Refer to [Set maximum segment size](/magic-wan/prerequisites/#set-maximum-segment-size) for more information.
+{{</Aside>}}
 
-In keeping with the example scenario, the list of static routes should match the example below.
+## 2. Add static routes
 
-![Static route configuration with defined prefixes, next hops, and priorities](/magic-wan/static/static-route-values.png)
+1. Follow the instructions in [Create a static route](/magic-wan/get-started/configure-static-routes/#create-a-static-route) to create the static routes.
```
Comment (angelampcosta) [inline content/magic-wan/tutorials/secure-web-gateway.md]:
```suggestion
1. [Create a static route](/magic-wan/get-started/configure-static-routes/#create-a-static-route).
```

**Review by angelampcosta — APPROVED** [review]

File: content/magic-wan/_partials/_traceroute.md (hunk 2 above)
Comment (marciocloudflare) [reply content/magic-wan/_partials/_traceroute.md:10]: Yup. That link goes to the how-to which has both GRE and IPsec options

**Pull request merged**